crates/daphne/src/protocol/mod.rs

```diff
@@ -81,18 +81,27 @@ mod test {
     use hpke_rs::HpkePublicKey;
     use prio::{
         codec::encode_u32_items,
-        field::Field64,
         vdaf::{
-            prio3::Prio3, AggregateShare, Aggregator as VdafAggregator, Collector as VdafCollector,
-            OutputShare, PrepareTransition,
+            prio3::Prio3, Aggregator as VdafAggregator, Collector as VdafCollector,
+            PrepareTransition,
+        },
+    };
+    use prio_draft09::{
+        field::Field64 as Field64Draft09,
+        vdaf::{
+            prio3::Prio3 as Prio3Draft09, AggregateShare as AggregateShareDraft09,
+            Aggregator as VdafAggregatorDraft09, Collector as VdafCollectorDraft09,
+            OutputShare as OutputShareDraft09, PrepareTransition as PrepareTransitionDraft09,
         },
     };
     use rand::prelude::*;
     use std::iter::zip;
 
     const TEST_VDAF: &VdafConfig = &VdafConfig::Prio3(Prio3Config::Count);
+    const TEST_VDAF_DRAFT09: &VdafConfig = &VdafConfig::Prio3Draft09(Prio3Config::Count);
 
     fn roundtrip_report(version: DapVersion) {
+        const CTX_STRING: &[u8] = "dap-13".as_bytes();
```

Comment:
nit: The "context string" is "dap-13" followed by the task ID. I'd refer to this as the context string prefix. However since this is test code, I would probably wait to construct the context string until the task ID is known. That is, on line 181 below:
```rust
let ctx = ["dap-13", &.task_id.0].concat();
```